### ISSUE TITLE
About incremental linking for object files on legacy macOS system to avoid "argument too long" dump

### ISSUE THREAD

**Issue by llv22** [bazelbuild/bazel#15746]
### Description of the feature request:

https://github.com/bazelbuild/bazel/issues/14530

### What underlying problem are you trying to solve with this feature?

1. params file doesn't support by the legacy libtool
2. too many object files during linking phase can easily lead to "arguments too long" issue on legacy system

### Which operating system are you running Bazel on?

macOS 10.13.6, Xcode 10.1

### Any other information, logs, or outputs that you want to share?

Basically I add the functionalities mentioned in https://github.com/llv22/bazel-macOS-10.13.6/releases/tag/5.2.1. So the building process can sail through on macOS 10.13.6, Xcode 10.1.
I'm not sure if your guys can consider this as a kind of feature for bazel. Otherwise, it's still a nightmare for developers on the legacy system to build monolitic library composed with a great amount of object files.
